media-sound/cadence/cadence-9999.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit git-r3 python-single-r1 xdg desktop
+
+DESCRIPTION="Collection of tools useful for audio production"
+HOMEPAGE="http://kxstudio.linuxaudio.org/Applications:Cadence"
+EGIT_REPO_URI="https://github.com/falkTX/Cadence.git"
+KEYWORDS=""
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="a2jmidid ladish -pulseaudio opengl"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
```

Comment:
I took this directly from the [upstream ebuild](https://gitweb.gentoo.org/repo/gentoo.git/tree/media-sound/cadence/cadence-9999.ebuild#n21). That said, I have looked into the real dependencies. For me, it looks like there are some build deps (for the c++ sources):
- QtGui
- QtCore
- QtWidgets
- QtSvg

Of course, there is the dependency to PyQt5 already, but I think, this is not enough, since Qt is used from C++ directly.

The rest should be RDEPENDs, yes.